Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -105,4 +105,9 @@ void Processor::check_invoke_scheduler()
     TODO_AARCH64();
 }
 
+StringView Processor::platform_string()
+{
+    return "aarch64"sv; // FIXME
```

Comment:
What's `FIXME` about this? should the string have more information?

Reply:
iʼll provide more details (it should be just extracted from registers). 🙇🏻‍♂️